### ISSUE TITLE
ISSN 2284-7308 | Perichoresis (Sciendo) | Translator

### ISSUE THREAD

**Issue by IxKo** [ubtue/DatenProbleme#2127]
#### URL
https://sciendo.com/issue/PERC/22/2
  
#### Import-Translator
  Einzel- und Mehrfachimport:
ubtue_Sciendo.js

  
### Problembeschreibung
Der Mehrfachimport funktioniert nicht und beim Einzel-Import tritt ein Fehler auf und es wird der allgemeine Embedded Metadata.js verwendet und nicht der auf uns angepasste ubtue.

(gemeldet von @IxTheoEg)

**Comment by mara-spiess** [comment]
@jriedl Bei mir lässt sich diese URL (und andere der Zeitschrift) leider gar nicht im Scaffold öffnen. Könnten Sie kurz testen ob sie das selbe Problem haben? Es spuckt auch keinerlei Fehlermeldung aus.

**Comment by jriedl** [comment]
Nein, lässt sich hier auch nicht öffnen. Im Browser erkennt man, dass beim Öffnen kurz eine Validierungslogik aufpopt, mit der der Scaffold-Browser wohl nicht umgehen kann. Workaround wäre wahrscheinlich wieder Speichern der Seite aus einem "richtigen" Browser und aufrufen der Datei.

**Comment by mara-spiess** [comment]
Das neue Update Zotero 7 scheint das öffnen der ZS nun möglich zu machen.

**Comment by mara-spiess** [comment]
Sowohl der Einzel- als auch der Mehrfachimport sollten nun wieder funktionieren.

**Comment by IxKo** [comment]
Problem besteht leider weiterhin bei mir.

**Comment by IxKo** [comment]
Danke funktioniert.

**Comment by IxKo** [comment]
Etwas zu früh geschlossen:
Der Mehrfachimport funktioniert zwar nun, aber dafür besteht das Problem beim Einzel-Import weiterhin, dass auf den Embedded Metadaten zurückgegriffen wird.

**Comment by mara-spiess** [comment]
Könnte der Einzelimport nochmal getestet werden? Die Fehlerquelle ist leider nicht einfach zu identifizieren.

**Comment by IxKo** [comment]
Einzel-Import klappt nun auch, danke.
